kits/isolate/apps/lib/runtime/daytona.ts

```
@@ -0,0 +1,787 @@
+          );
+          if (cleanup.exitCode === 0) {
+            cleaned = true;
+            break;
+          }
+          cleanupError = new Error("Probe artifact cleanup failed.");
+        } catch (error) {
+          cleanupError = error;
+        }
+        console.error("Isolate probe artifact cleanup failed", cleanupError);
+      }
+      if (!cleaned) throw cleanupError;
```

Biome (2.5.5)

[error] Unsafe usage of 'throw'.

(lint/correctness/noUnsafeFinally)